src/eventDispatchers/mouseEventHandlers/mouseDown.js

```diff
@@ -2,12 +2,13 @@ import external from './../../externalModules.js';
 // State
 import { getters, state } from './../../store/index.js';
 import { getToolState } from './../../stateManagement/toolState.js';
-// Manipulators
-import getHandleNearImagePoint from './../../manipulators/getHandleNearImagePoint.js';
-import moveAllHandles from './../../manipulators/moveAllHandles.js';
-import moveHandle from './../../manipulators/moveHandle.js';
 // Util
 import isMouseButtonEnabled from './../../util/isMouseButtonEnabled.js';
+import {
+  getToolsWithMovableHandles,
+  findHandleDataNearImagePoint,
+  findAnnotationNearClick
+} from '../../base/baseAnnotationToolHelpers.js';
```

Comment:
If we're going to use these here, maybe a different shared/file name?

Reply:
Yeah you are correct, we should probably go on a renaming frenzy nearer completion of all the tools.